src/features/comments.ts

```diff
@@ -0,0 +1,75 @@
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../types/Comment';
+import { createComment, deleteComment, getPostComments } from '../api/comments';
+
+type CommentsState = {
+  comments: Comment[];
+  loaded: boolean;
+  error: boolean;
+};
+
+const initialState: CommentsState = {
+  comments: [],
+  loaded: false,
+  error: false,
+};
+
+export const fetchComments = createAsyncThunk(
+  'comments/fetch',
+  (postId: number) => getPostComments(postId),
```

Comment:
Ensure that the `getPostComments` function is correctly implemented and handles errors properly. This will help prevent issues when fetching comments.